src/components/cvaa-overlay/cvaa-overlay.tsx

```diff
@@ -131,14 +131,17 @@ class CVAAOverlay extends Component<any, any> {
    */
   render(props: any): VNode<any> {
     props.clearAccessibleChildren();
+    const isMainOverlay = this.state.activeWindow === cvaaOverlayState.Main;
+    const ariaProps = isMainOverlay ? { ariaLabelledBy: 'captionsDialogTitle' } : { ariaLabel: 'Custom Captions Settings' };
```

Comment:
We usually don't use hardcoded english texts.
Is this a text that doesn't need translation ? And non of the existing cvaa localized texts can be used here.

If this text is new, and doesn't exist anywhere yet in the ui (so it's not possible to reuse the localized version), it should be added to an en.18n.json file under "cvaa" section, and product should be notified that this text requires localization, so it can be translated in the next opportunity.

For examples of localization you can look at uses of withText decorator in playkit-ui.